presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -311,6 +312,10 @@ public static void setReadColumns(Configuration configuration, List<Integer> rea
             return MapredParquetInputFormat.class;
         }
 
+        if (PageInputFormat.class.getSimpleName().equals(inputFormatName)) {
```

Comment:
Could you please remind me why this change is needed?

Reply:
> Could you please remind me why this change is needed?

Otherwise, the line after this will fail :  `Class<?> clazz = conf.getClassByName(inputFormatName);`